src/components/layouts/Classic/README.md

```diff
@@ -0,0 +1,6 @@
+# Layouts
+
+## Classic
+
+> TODO: add few lines explaining the use case, features and purpose of this layout, like the
```

Comment:
@ancorgs as discussed, a few lines from you explaining how and why this layout is supposed to work, are very welcome (now or in the future)

Everyone else: a better name for the layout will be great too.